### DISCUSSION TITLE
Igx-financial-charts : Is there a way to add a dialog when clicked on a particular callout if there are different callouts at at one particular index

### DISCUSSION THREAD

**Discussion by amitmoondra** [IgniteUI/igniteui-angular#12483] [Q&A]
## Question  
1. 
Is there a way to set different designs for callouts, so in my case I have multiple callouts at a particular time in one series.
I want the ability to give them separate CSS styling.
Currently, I can only give them a different label.

2. 
Is there a way I can show different dialog when hovering/clicked over different callouts, 
this essentially is needed for me to give users the ability to get into more brief info of the callouts and also give a link for redirection.

Please let me know your input.

**Comment by mddifilippo89** [comment]
Hello, 
1)
I suggest using the [IgxDataChart with the Financial Series](https://github.com/IgniteUI/igniteui-angular-examples/tree/vnext/samples/charts/data-chart/financial-price-series). You may already be doing this if you are using multiple callouts. Please clarify if you are or not. In order to style individual callouts you'll need to separate out each callout layer with its own datasource. See attached. 
[axis-annotations-corner-radius.zip](https://github.com/IgniteUI/igniteui-angular/files/10043574/axis-annotations-corner-radius.zip)

2) Hook the chart's seriesMouseLeftButtonDown and it will provide unique details on the clicked callout compared to the underlying item. 

eg.

public ngAfterViewInit(): void {

        if (this.chart !== undefined ) {
            this.chart.seriesMouseLeftButtonDown.subscribe((e: IgxDataChartMouseButtonEventArgs) =>
                this.onSeriesMouseLeftButtonDown(this.chart, e)
            );
        }
    }

    public onSeriesMouseLeftButtonDown(chart: IgxDataChartComponent, args: IgxDataChartMouseButtonEventArgs){
        let v = args.item as CountryRenewableElectricityItem;
        console.log(args);
    }

**Comment by mddifilippo89** [comment]
From sample attached

![image](https://user-images.githubusercontent.com/17785923/202777504-7de47096-6bcf-4c8a-8cda-035e8bf6fbe5.png)

**Comment by amitmoondra** [comment]
Checking this now, will let you know if this solves my purpose.
